turbo/snapshotsync/freezeblocks/block_snapshots.go

```diff
@@ -215,13 +215,18 @@ func (br *BlockRetire) borSnapshots() *heimdall.RoSnapshots {
 	return br.blockReader.BorSnapshots().(*heimdall.RoSnapshots)
 }
 
-func CanRetire(curBlockNum uint64, blocksInSnapshots uint64, snapType snaptype.Enum, chainConfig *chain.Config) (blockFrom, blockTo uint64, can bool) {
+func CanRetire(curBlockNum uint64, blocksInSnapshots uint64, maxAvailableBlockInDb uint64, snapType snaptype.Enum, chainConfig *chain.Config) (blockFrom, blockTo uint64, can bool) {
```

Comment:
`curBlockNum` seems same thing as `maxAvailableBlockInDb` (progress of stage_senders)